libraries/botbuilder-dialogs/tests/test_skill_dialog.py

```diff
@@ -148,10 +148,13 @@ async def capture(
             conversation_state=conversation_state,
         )
 
+        assert len(dialog_options.conversation_id_factory.conversation_refs) == 0

```

Comment:
this is nitpicking but this could be changed to 
`assert not dialog_options.conversation_id_factory.conversation_refs`